lanelet2_python/python_api/geometry.cpp

```diff
@@ -289,8 +290,21 @@ BOOST_PYTHON_MODULE(PYTHON_API_MODULE_NAME) {  // NOLINT
   def("area", boost::geometry::area<ConstHybridPolygon2d>);
 
   class_<ArcCoordinates>("ArcCoordinates", "Coordinates along an arc", init<>())
+      .def("__init__", make_constructor(
+                           +[](double length, double distance) {
+                             return boost::make_shared<ArcCoordinates>(ArcCoordinates{length, distance});
+                           },
+                           default_call_policies(),
+                           (arg("length") = 0., arg("distance") = 0.)))
```

Comment:
The default constructor is now is now superfluous with the new constructor. And boost also understands std::shared_ptr:
```suggestion
  class_<ArcCoordinates>("ArcCoordinates", "Coordinates along an arc", no_init)
      .def("__init__", make_constructor(
                           +[](double length, double distance) {
                             return std::make_shared<ArcCoordinates>(ArcCoordinates{length, distance});
                           },
                           default_call_policies(),
                           (arg("length") = 0., arg("distance") = 0.)))
```

Reply:
Sounds good, I am not very experienced with boost, just trying to understand and copy from the existing interface code.